pkg/commands/compute/validate.go

```diff
@@ -49,6 +49,14 @@ func (c *ValidateCommand) Exec(_ io.Reader, out io.Writer) error {
 		return fmt.Errorf("error reading file path: %w", err)
 	}
 
+	manifestFilename := manifest.Filename
+	if c.env != "" {
+		manifestFilename = fmt.Sprintf("fastly.%s.toml", c.env)
+		if c.Globals.Verbose() {
+			text.Info(out, "Using the '%s' environment manifest (it will be packaged up as %s)\n\n", manifestFilename, manifest.Filename)
```

Comment:
Should we print this outside of the c.env if statement?

Reply:
We _could_ do. The feedback I've typically had from users has been the CLI is too noisy.

But on the flip-side I've also had users say things like "why do I have to use --verbose to see certain information".

It's all a balance and we won't be able to keep everyone happy unfortunately.

**That said, in this particular case my thinking was along these lines...**

Setting the `--env` flag is something the user knows they've done, because they've explicitly set it when calling the CLI command (unless they're using some kind of custom abstraction around the CLI and someone has added it as part of that custom abstraction). So if the user has had to explicitly type `--env` as part of the command invocation, then it should be reasonable to presume they know what they've set and why, and that's why I decided, in this case, to keep the extra INFO message behind the `--verbose` flag.